### PR TITLE
INSTUI-3311 - new Pagination variant with NumberInput

### PR DESCRIPTION
**Pull request by ke1k0**

### CODE REVIEW

**Review by Brailor — APPROVED** [review]
Looks good!

File: packages/ui-pagination/src/Pagination/PaginationPageInput/__tests__/PaginationPageInput.test.tsx [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+      const pageInput = await PaginationPageInputLocator.find()
+      const input = await pageInput.findInput()
+
+      await input.keyDown('down')
+
+      expect(input.getDOMNode().value).to.equal('3')
+      expect(onChange).to.have.been.calledWithMatch(match.object, 2)
+    })
+  })
+})
```
Comment (matyasf) [inline packages/ui-pagination/src/Pagination/PaginationPageInput/__tests__/PaginationPageInput.test.tsx:368]: nice test suite :)

**Review by matyasf — APPROVED** [review]
looks good :)

**Review by HerrTopi — APPROVED** [review]
Seems nice, seem to be working nice. Good work!

**Pull request merged**